### ISSUE TITLE
UI: Customize Argo CD UI colors by overriding colors.scss variables with a configmap

### ISSUE THREAD

**Issue by rodrigo14miguel** [argoproj/argo-cd#11249]
# Summary

Add the ability to easily customize Argo CD UI colors by overriding the variables defined in argo-ui "v2/styles/colors.scss".

# Motivation

The current option is to use [Custom Styles](https://argo-cd.readthedocs.io/en/stable/operator-manual/custom-styles/) but this requires CSS knowledge. 
It would be nice to have a simple way to edit the UI colors only.

# Proposal

It seems that most colors used in the UI are declared in [v2/styles/colors.scss](https://github.com/argoproj/argo-ui/blob/master/v2/styles/colors.scss) of argo-ui project (not all it seems, can't find Argo CD v2.5 sidebar colors in there).
It would be nice to use a configmap to override the color values in this file.

**Comment by Marvin9** [comment]
Might be similar to https://github.com/argoproj/argo-cd/issues/9754

I don't think it is possible to override scss as its declarations are lost once UI build is done.

**Comment by rodrigo14miguel** [comment]
Hi @Marvin9 
yes it does seam similar. ill close this one in favor of #9754
